v3iofs/fs.py

```diff
@@ -53,7 +53,7 @@ def get(self, key):
         if lookup_result is None:
             return None
         key_time, value = lookup_result
-        if key_time <= start_time + self._cache_validity_seconds:
+        if start_time <= key_time + self._cache_validity_seconds:
```

Comment:
1. at least for me (as a newcomer) the "start_time" name was not self-explanatory. how about "time_of_get" or similar?
2. moving the start_time just before the if clause is more accurate and can save some cycles in case there was a cache miss.

Reply:
The whole point of the name and position is that it's at the start of the method. Is it necessary though? Seems to me like it's not.